spiffetls/src/c_dial_example.c

```diff
@@ -38,11 +45,25 @@ int main(void)
                                        /*127.0.0.1*/ (in_addr_t) 0x7F000001,
                                        mode, &config, &err);
 
+    if(conn == NULL) {
+        printf("spiffetls_DialWithMode() failed\n");
+        exit(-1);
+    }
+
     if(err != NO_ERROR) {
-        printf("could not create TLS connection!");
+        printf("could not create TLS connection\n");
         exit(-1);
     }
 
+    char buff[1024];
+    int bytes;
+    char *message = "Hi Server!";
```

Comment:
It would be nice if the message could be passed as an argument when running the example

Reply:
done